test/exhal/form_test.exs

```diff
@@ -9,6 +9,13 @@ defmodule ExHal.FormTest do
   import Mox
   setup :verify_on_exit!
 
+  setup do
+    Application.put_env(:exhal, :client, ExHal.ClientMock)
+    on_exit fn ->
+      Application.put_env(:exhal, :client, ExHal.Client)
```

Comment:
When running `mix test test/` if the client is not returned to `ExHal.Client` it fails tests downstream.